src/domain/auth/authenticate.ts

```diff
@@ -10,9 +10,13 @@ import {
 import { TUNNISTAMO_API_TOKEN_ENDPOINT } from '../api/constants/ApiConstants';
 import { BackendTokenResponse } from './types/BackendAuthenticationTypes';
 
-export default function(path: string): void {
+export const loginToTunnistamo = (path?: string) => {
```

Comment:
Naming is not consistent with `logoutTunnistamo` 

What if we call them: `loginToTunnistamo` and `logoutFromTunnistamo` or just `loginTunnistamo` and `logoutTunnistamo`

or (my favourite) `loginViaTunnistamo` & `logoutViaTunnistamo`?

Reply:
loginTo and loginFrom are more cute i would say